vscode-car-plugin/src/main/java/org/wso2/maven/libraries/ConnectorDependencyResolver.java

```diff
@@ -78,10 +85,14 @@ public static void resolveDependencies(CARMojo carMojo) throws Exception {
             }
         }
 
+        // Scan local entries folder for connections
+        Map<String, String> connectionTypeMap = scanLocalEntriesForConnections(Constants.LOCAL_ENTRIES_FOLDER_PATH,
```

Comment:
Shall we do this check only if connectionType is provided in the descriptor file?